### ISSUE TITLE
Compile error

### ISSUE THREAD

**Issue by VZIKL** [CleverRaven/Cataclysm-DDA#22448]
Game version:0.C-25055-gb94c476f73 / Git:master

Operating system:Gentoo

Tiles or curses:Tiles
Compile arguments:
make -j4 CCACHE=1 CLANG=1 TILES=1 SOUND=1 NATIVE=linux64 LUA=1 all
Error message:
src/sdltiles.cpp:1562:11: error: expected ';' after expression
    stdscr newwin(get_terminal_height(), get_terminal_width(),0,0);

Maybe i should add a equals sign ?
->stdscr = newwin(get_terminal_height(), get_terminal_width(),0,0);

**Comment by aeoo** [comment]
>Maybe i should add a equals sign ?

I did just that and it fixed the compilation for me.  The game appears to work fine as well after the fix.  Also, I looked at the rest of the @BevapDin 's commit and I noticed there was another place that mentioned `stdscr` and that other place had an assignment operator (equals sign), so I am 90% sure that yea, it's just a typo, and an equals sign is supposed to be in that line you have pointed out.
